fms/utils/serialization.py

```diff
@@ -244,6 +253,7 @@ def load_state_dict_into_model(
     state_dict: MutableMapping[str, Any],
     architecture: str,
     source: str,
+    model_config: Optional[ModelConfig] = None,
```

Comment:
since there are getting to be so many params and easy to miss-order them, I'm starting to think maybe this function should be kwargs-only too